README.md

```diff
@@ -79,7 +81,7 @@ Download the [custom-resource-stack.yaml](https://github.com/aws/aws-auto-scalin
 Run the following [create-stack](https://docs.aws.amazon.com/cli/latest/reference/cloudformation/create-stack.html) command, adding your details to the following parameters:
 
 1. *SNSSubscriptionEmail*: Replace *email-address* with an email address to send certificate expiry notifications to.
-2. *IntegrationHttpEndpoint*: Replace *endpoint-url* with your REST endpoint URL, for example, http://api.example.com/v1/scalableTargetDimensions/1-23456789.
+2. *IntegrationHttpEndpoint*: Replace *endpoint-url* with your REST endpoint URL, for example, http://api.example.com/v1/scalableTargetDimensions/{scalableTargetDimensionId}
```

Comment:
I'd let Renee comment on this. She suggested we put some dummy values instead of the placeholder

Reply:
I made this change because I found it confusing.
Maybe include both?
http://api.example.com/v1/scalableTargetDimensions/{scalableTargetDimensionId}
Where {scalableTargetDimensionId} is replaced with the dimension in your backend API, like:
http://api.example.com/v1/scalableTargetDimensions/1-23456789

Reply:
may be. I am okay with it